.azure-pipelines/ipu-tests.yml

```diff
@@ -72,26 +72,11 @@ jobs:
         python -c "import poptorch; print(poptorch.__version__)"
       displayName: "Check poptorch installation"
 
-    - bash: |
-        wget https://pl-public-data.s3.amazonaws.com/legacy/checkpoints.zip -P legacy/
-        unzip -o legacy/checkpoints.zip -d legacy/
-        ls -l legacy/checkpoints/
-      displayName: 'Get legacy checkpoints'
-
     - bash: |
         source ${{ variables.poplar_sdk }}/poplar-ubuntu*/enable.sh
         source ${{ variables.poplar_sdk }}/popart-ubuntu*/enable.sh
         export POPTORCH_WAIT_FOR_IPU=1
-        python -m coverage run --source pytorch_lightning -m pytest pytorch_lightning tests -v --junitxml=$(Build.StagingDirectory)/test-results.xml --durations=50
```

Comment:
I don't think we need to run the full test suite on the IPU machine for now, here's why:

- There is only 1 IPU job available, so this is a huge bottleneck
- The IPU machine CPU is quite slow compared to the GPU one, so non-accelerator tests run considerably slower
- IPUs are experimental. By reducing the number of things this job tests, it's more feasible for us to make this a required check.

Reply:
Do we lose much IPU coverage by disabling the other tests, or are the disabled tests all run without an accelerator backend enabled?

Reply:
For the actual IPU tests we lose none as all of them have been included in `pytest tests/accelerators/test_ipu.py` which is now the only file run in IPU testing after this PR is merged!

Reply:
Great!